dev_scripts/exportTranslation.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable default-case */
+/* eslint-disable array-callback-return */
+/* eslint-disable func-names */
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+/* eslint-disable no-console */
+/* eslint-disable no-useless-escape */
+/* eslint-disable no-shadow */
+/* eslint-disable guard-for-in */
+/* eslint-disable prefer-const */
+/* eslint-disable no-undef */
+/* eslint-disable no-restricted-syntax */
+
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+const fs = require('fs');
+
+const localizationFiles = [];
+let index = 0;
+
+// Define mainLanguage as English (gb)
+const mainLanguage = 'gb';
+
+getLocalizationFiles = () => {
+  // Reads all files in localization files and save them in localizationFiles array
+  fs.readdirSync('././src/localization/').forEach(file => {
+    // Filter to only get JSON files
+    if (file.split('.').pop() === 'json') {
+      const fileName = require(`../src/localization/${file}`);
+      localizationFiles.push(fileName);
+      localizationFiles[index].name = file.replace('.json', '');
```

Comment:
one of the cool things about using new ES standards is destructuring of object/array:
`localizationFiles.push({...fileContent, fileName: file.replace('.json', '')})`
The way you've written above is fine btw, but JS structure would be more readable if you:
`localizationFiles.push({fileContent: require(`../src/localization/${file}`), fileName: file.replace('.json', '')})

Reply:
Great advice, I reduced so many lines with this!